### PR TITLE
i18n - pulling from tx

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: locale/fr/foreman_bootdisk.po [hunk 1]
```diff
@@ -256,7 +256,7 @@ msgid "Unable to generate disk %{kind} template: %{error}"
 msgstr "Impossible de générer le modèle %{kind} de disque : %{error}"
 
 msgid "Unable to generate disk template, %{kind} template not found."
-msgstr "Impossible de générer le modèle de disque, le modèle %%{kind} est introuvable."
+msgstr "Impossible de générer le modèle de disque, le modèle%{kind} est introuvable."
```
Comment (ekohl) [inline locale/fr/foreman_bootdisk.po:259]: This looks like a regression. I've just applies this change to Transifex so if you pull again it should be good again
```suggestion
msgstr "Impossible de générer le modèle de disque, le modèle %{kind} est introuvable."
```

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by stejskalleos** [comment]
Translations updated

**Review by ekohl — CHANGES_REQUESTED** [review]
When updating translations, you need to make sure the `.mo` is also updated because that gets used at runtime.

**Comment by stejskalleos** [comment]
> When updating translations, you need to make sure the .mo

I run precisely the same steps as I do with other plugins, but the `.mo` files are just not updated.

**Pull request closed without merging**